spec/support/corpus.rb

```diff
@@ -58,18 +66,16 @@ def verify_mutation_coverage
       #
       # @raise [Exception]
       #   otherwise
-      #
-      # rubocop:disable AbcSize
       def verify_mutation_generation
         checkout
         start = Time.now
-        paths = Pathname.glob(repo_path.join('**/*.rb')).sort_by(&:size).reverse
+
         options = {
```

Comment:
Also not changed, but I wondered why the `options` isn't a singleton method, and why it's not frozen.